extensions/amp-accordion/1.0/example/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://cdn.ampproject.org/bento.js"></script>
+    <script
+      async
+      src="https://cdn.ampproject.org/v0/bento-accordion-1.0.js"
+    ></script>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.ampproject.org/v0/bento-accordion-1.0.css"
+    />
+  </head>
+  <body>
+    <body>
+      <bento-accordion id="my-accordion">
+        <section>
+          <h2>Section 1</h2>
+          <p>Content in section 1.</p>
```

Comment:
i think you need to rebase against `main` - i had change to convert the first `<section>` body in the accordion examples to use a `<div>` instead of a `<p>` to be consistent with the other sections

Reply:
i think the amp-accordion changes are completely missing now?

Reply:
Yes, [because it's already present](https://github.com/ampproject/amphtml/blob/main/extensions/amp-accordion/1.0/example/index.html)